colin/core/config/fedora.json

```diff
@@ -2,7 +2,8 @@
   "labels": {
     "required": [
       "maintainer",
-      "name"
+      "name",
+      "com_redhat_component"
```

Comment:
I'm lazy to go through the complete source code; please enlighten me how this gets expanded into `com.redhat.component`.

Reply:
Yes, you're right. There is a problem with the naming:

There are two check names -- file and name property. It would be nice if they can be same.

Reply:
It has to be corrected in the other checks as well.

Reply:
Could you please open an issue so we can track this?

Reply:
#6